src/app/modules/auth/auth.controller.ts

```diff
@@ -311,7 +309,7 @@ export const getRedirectLink: RequestHandler<
 export const handleRedirect = [
   celebrate({
     [Segments.QUERY]: Joi.object({
-      isPersistentLogin: Joi.boolean().optional(),
+      isPersistentLogin: Joi.boolean(),
```

Comment:
this is a backwards incompatible change, I'd suggest reverting this.

Reply:
sure. i'll still leave the frontend with default parameters but will revert this change to make it backwards compatible

Reply:
Will need to revert the `!!` change since now there is a possibility for undefinedness